src/data-root.ts

```diff
@@ -79,11 +79,12 @@ export async function update(
   const apiEndpoint = setup.endpoints.api
 
   // Debug
-  debug.log("🚀 Updating your DNSLink:", cid)
+  debug.log("🌊 Updating your DNSLink:", cid)
```

Comment:
v. important change. 👍 😉

Reply:
Well, we can't have a proper release if the emojis aren't in the same theme 😬